tests/src/kinClient.int.test.ts

```diff
@@ -0,0 +1,158 @@
+import {Server} from "@kinecosystem/kin-sdk";
+
+import {KinAccount} from "../../scripts/src/kinAccount";
+import {AccountDataRetriever} from "../../scripts/src/blockchain/accountDataRetriever";
+import {Environment} from "../../scripts/src/environment";
+import {Channel, ChannelsPool, KeyPair, KinClient} from "../../scripts/src";
+import {Friendbot} from "../../scripts/src/friendbot";
+import {TransactionBuilder} from "../../scripts/src/blockchain/transactionBuilder";
+
+const url = Environment.Testnet.url;
+const server = new Server(url, {allowHttp: true});
+const keypair = KeyPair.generate();
+const seconedKeypair = KeyPair.generate();
+let client: KinClient;
+let friendBot: Friendbot;
+let account: KinAccount;
+let account2: KinAccount;
+
+describe("KinAccount.createAccount", async () => {
+	beforeAll(async () => {
+		client = new KinClient(Environment.Testnet)
+		friendBot = new Friendbot(url, new AccountDataRetriever(server));
+		account = client.createKinAccount({seed: keypair.seed});
+		account2 = client.createKinAccount({seed: seconedKeypair.seed});
+		await client.friendbot({address: keypair.publicAddress, amount: 10000});
+		await client.friendbot({address: seconedKeypair.publicAddress, amount: 10000});
```

Comment:
add verification (expect) to the creation of the accounts so it will fail immediately with no success, and so it will be more clear what is the failure.